SQL/2015-03-9-Notification_table_New_Columns.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE notification_spool ADD COLUMN `Origin` varchar(255) DEFAULT NULL;
+ALTER TABLE notification_spool ADD COLUMN `ProcessID` int(11) NOT NULL DEFAULT '0' AFTER NotificationTypeID;
+ALTER TABLE notification_spool CHANGE COLUMN TimeSpooled TimeSpooled datetime DEFAULT NULL;
```

Comment:
Does this automatically convert the existing data from epoch time to SQL datetime? If not, the patch needs to handle that for existing projects. It might be better/safer to just add a new column and transition to using that one.